Packs/FortiGate/ReleaseNotes/1_0_24.md

```diff
@@ -0,0 +1,3 @@
+#### Modeling Rules
+##### Fortinet FortiGate Modeling Rule
+Updated the modeling rule with additional fields.
```

Comment:
Perhaps you should list the rules that were added?

Reply:
@ShirleyDenkberg 
I didn't add rules, but fields. It is a pretty long list to add for the release note so maybe should I rephrase it?

Reply:
The description is clear.  I understand that you added field (my mistake in my comment). I guess you can leave it as it.